esmvaltool/preprocessor/__init__.py

```diff
@@ -48,6 +48,7 @@
     'extract_time',
     'extract_season',
     'extract_month',
+    'annual_mean',
```

Comment:
I think this should be done later in the preprocessor order, see https://esmvaltool.readthedocs.io/en/version2_development/user_guide2/index.html#architecture
@mattiarighi Do you have a suggestion?